.circleci/config.yml

```diff
@@ -44,106 +69,87 @@ jobs:
           environment:
             KEYSTONE_SDK_DIR: /keystone/sdk/build
           command: |
-              make format
+              FORMAT=$(git help -a | grep clang-format | tail -n1)
```

Comment:
nit: Please add comments on what this line is doing

Reply:
Added a comment describing this